features/standard/ContentTranslation.feature

```diff
@@ -0,0 +1,44 @@
+@javascript @translation @IbexaOSS @IbexaContent @IbexaExperience @IbexaCommerce
+Feature: Content item transation
+
+  @javascript @APIUser:admin
+  Scenario: Publish new translation based on existing translation
+    Given I create "folder" Content items in root in "eng-GB"
+      | name             | short_name       | short_description | description      |
+      | EnglishPublished | EnglishPublished | EnglishPublished  | EnglishPublished |
+    And I am logged as admin
+    And I'm on Content view Page for "EnglishPublished"
+    When I switch to "Translations" tab in Content structure
+    And I add new translation "French" basing on "English (United Kingdom)" translation
+    And I set content fields
+      | label      | value           |
+      | Name       | FrenchPublished |
+      | Short name | FrenchPublished |
+    And I click on the edit action bar button "Publish"
+    Then success notification that "Content published." appears
+    And I choose "French" preview
```

Comment:
```suggestion
    And I choose "French" preview in Content View
```

I'd add this to be more specific, in case we will be dealing with other previews in the future